<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D房屋绘制</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_set">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>313</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>参数设定</string>
    </property>
    <widget class="QGroupBox" name="groupBox_projection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>投影类型</string>
     </property>
     <widget class="QRadioButton" name="radioButton_perspective">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>透视投影</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_parallel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>平行投影</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_vrp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>视图参考点VRP(WC)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>z：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_vrpx">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_vrpy">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_vrpz">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>视图参考坐标系偏转角</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>theta：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>phi：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>delta：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_theta">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_phi">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_delta">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_prp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>投影参考点PRP（VRC）</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>n：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prpu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>u：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>v：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prpv">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prpn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>窗口中心CW（VRC）</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>u：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cwv">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>v：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cwu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_apply">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>应 用</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_restore">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>500</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>恢 复</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="layoutWidget" name="widget_1" native="true"/>
    <widget class="projectionWidget" name="widget_2" native="true"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>layoutWidget</class>
   <extends>QWidget</extends>
   <header location="global">layoutwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>projectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">projectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
